src/common/styles/themes.tsx

```diff
@@ -49,7 +55,7 @@ export type CustomTheme = {
     backgroundColor: string;
     thumbsUpIconColor: string;
   };
-  checkoutUnsuccessfulCard: {
+  checkoutUnsuccessCard: {
```

Comment:
Is there a better way to split the theme? maybe like having
```
export type SharedTheme = { ... }
export type SallyTheme = { ... }
export type GovwalletTheme = { ... }
```

Reply:
do you mean in terms of typing? currently both the sallyTheme and govWalletTheme are typed as `CustomTheme`!

Reply:
oh I didnt see the above and below 😂